src/Compilers/CSharp/Test/Symbol/Symbols/Source/UsingAliasTests.cs

```diff
@@ -969,5 +970,104 @@ void M()
                 }
                 """).VerifyDiagnostics();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68546")]
+        public void AliasToArrayReferencedInUsingStatic()
+        {
+            CreateCompilation("""
+                using A = int[];
+
+                namespace N
+                {
+                    using static A;
+                }
+                """).VerifyDiagnostics(
+                    // (5,5): hidden CS8019: Unnecessary using directive.
+                    //     using static A;
+                    Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using static A;").WithLocation(5, 5),
+                    // (5,18): error CS9137: 'array' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate or namespace can be used.
+                    //     using static A;
+                    Diagnostic(ErrorCode.ERR_BadUsingStaticType, "A").WithArguments("array").WithLocation(5, 18));
+        }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68546")]
+        public void AliasToPointerReferencedInUsingStatic()
+        {
+            CreateCompilation("""
+                using unsafe A = int*;
+
+                namespace N
+                {
+                    using static A;
+                }
+                """, options: TestOptions.UnsafeDebugDll).VerifyDiagnostics(
+                    // (5,5): hidden CS8019: Unnecessary using directive.
+                    //     using static A;
+                    Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using static A;").WithLocation(5, 5),
+                    // (5,18): error CS9137: 'pointer' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate or namespace can be used.
+                    //     using static A;
+                    Diagnostic(ErrorCode.ERR_BadUsingStaticType, "A").WithArguments("pointer").WithLocation(5, 18),
+                    // (5,18): error CS0214: Pointers and fixed size buffers may only be used in an unsafe context
```

Comment:
>// (5,18): error CS0214: Pointers and fixed size buffers may only be used in an unsafe context [](http://example.com/codeflow?start=20&length=94)

Consider suppressing this error to reduce the noise.

Reply:
sure.  let me see how possible that is.